docs/src/examples/solvers/Path planning sgmpc.jl

```diff
@@ -57,7 +57,7 @@ concrete_system = concrete_problem.system;
 
 # Definition of the grid of the state-space on which the abstraction is based (origin `x0` and state-space discretization `h`):
 x0 = SVector(0.0, 0.0, 0.0);
-h = SVector(0.2, 0.2, 0.2);
+h = SVector(0.1, 0.1, 0.2);
```

Comment:
@johnaoga cc my e-mail, just revert if you want to go back to the coarser abstraction

Reply:
Thank you! No worries!